src/routes/api/auth/logout/+server.ts

```diff
@@ -0,0 +1,33 @@
+import { JWT_ACCESS_KEY, JWT_REFRESH_KEY } from '$lib/auth';
+import type { RequestHandler } from '@sveltejs/kit';
+
+export const POST: RequestHandler = async (event) => {
+  const accessToken = event.cookies.get(JWT_ACCESS_KEY);
+  const refreshToken = event.cookies.get(JWT_REFRESH_KEY);
+  if (!accessToken || !refreshToken) {
+    return new Response(
+      JSON.stringify({
+        error: 'You are not logged in'
+      }),
+      { status: 401 }
+    );
+  }
+
+  const response = new Response(
+    JSON.stringify({
+      message: 'You have been logged out'
+    }),
+    {
+      status: 200
+    }
+  );
```

Comment:
One could think about abstracting these responses but we'll probably never do something like this again so I don't think it's worth it. Still dropping the idea though

Reply:
I'll subscribe to what you just said, understand but do not think it's worth it given the use case